pkgs/development/libraries/frei0r/default.nix

```diff
@@ -5,7 +5,7 @@ stdenv.mkDerivation rec {
   version = "1.4";
 
   src = fetchurl {
-    url = "https://files.dyne.org/frei0r/releases/frei0r-plugins-1.4.tar.gz";
+    url = "https://files.dyne.org/frei0r/releases/${name}.tar.gz";
```

Comment:
I initially had to fix this url which had changed.
I also avoided repetition by using the `name`.
After rebase, I solve the conflict letting only the factorisation appear.
I can remove this commit if you see this as inconvenient commit.